hudi-common/src/main/java/org/apache/hudi/avro/HoodieAvroUtils.java

```diff
@@ -1499,21 +1503,21 @@ private static Schema getActualSchemaFromUnion(Schema schema, Object data) {
   public static HoodieRecord createHoodieRecordFromAvro(
       IndexedRecord data,
       String payloadClass,
-      String preCombineField,
+      Option<String[]> preCombineFields,
```

Comment:
Should we have this be an empty array if there are no ordering fields?

Reply:
We are setting it to default ordering value if no preCombine fields are available. 
`org.apache.hudi.common.util.SpillableMapUtils#getPreCombineVal`